bots/helpers/live_chat.py

```diff
@@ -0,0 +1,136 @@
+"""Modules for interacting with the liveChatMessages api"""
+# pylint: disable=too-many-arguments
+import asyncio
+import logging
+import os
+import requests
+from dotenv import load_dotenv
+from helpers.musicplayer import mpdproxy
+from helpers.postgres import postgresproxy
+
+async def get_live_chat_messages(youtube, live_chatid):
+    """Get YouTube Live Chat Messages"""
+    logger = logging.getLogger('YoutubeProBot')
+
+    try:
+        next_page_token = ''
+
+        while True:
+            if not next_page_token:
+                request = youtube.liveChatMessages().list(
+                    liveChatId=live_chatid,
+                    part="snippet,authorDetails",
+                    maxResults=1000
+                )
+                response = request.execute()
+                logger.debug("Get LiveChatMessage list - Success!")
+
+            else:
+                request = youtube.liveChatMessages().list(
+                    liveChatId=live_chatid,
+                    part="snippet,authorDetails",
+                    maxResults=1000,
+                    pageToken=next_page_token
+                )
+                response = request.execute()
+                logger.debug("Get LiveChatMessage NextPage list - Success!")
+
+            if response:
+                await discover_command_requests(youtube, response, live_chatid)
+
+            next_page_token = response["nextPageToken"]
+            wait_time = response["pollingIntervalMillis"]/1000
```

Comment:
Is Millis a common abbreviation for Millisecond (Msec)?

Reply:
lol love how you put the more known abbreviation in the question. I have seen millsec but millis is weird.

pollingIntervalMillis is from the youtubeapi so we have no control over this name unfortunately.